src/core/MIGRATION.md

```diff
@@ -824,6 +825,46 @@ After plugins, core is where most of the rest of the code in Kibana will exist.
 
 The packages directory should have the least amount of code in Kibana. Just because some piece of code is not stateful doesn't mean it should go into packages. The packages directory exists to aid us in our quest to centralize as many of our owned dependencies in this single monorepo, so it's the logical place to put things like Kibana specific forks of node modules or vendor dependencies.
 
+### How do I build my shim for New Platform services?
+
+Many of the utilities you're using to build your plugins are available in the New Platform or in New Platform plugins. To help you build the shim for these new services, use the tables below to find where the New Platform equivalent lives.
+
+#### Client-side
+
+In client code, `core` can be imported in legacy plugins via the `ui/new_platform` module.
+
+```ts
+import { npStart: { core } } from 'ui/new_platform';
+```
+
+| Legacy Platform                                       | New Platform                   | Notes                                                                                                                                          |
+|-------------------------------------------------------|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `chrome.addBasePath`                                  | `core.http.basePath.prepend`   | This API is also available on the server                                                                                                       |
```

Comment:
I'd add it to server-side table and remove the note. It's easy to miss when you scan content of the tables